icubam/predicu/plot/operational_dashboard.py

```diff
@@ -0,0 +1,204 @@
+
```

Comment:
This should be put into a different folder, maybe ` icubam/predicu/dashboard`. All modules in ` icubam/predicu/plot/` are assumed to be plots with a specific API (e.g. have a plot function that returns something) and are automatically tested by pytest.

Reply:
Ah good point!

Reply:
cool! done